hwpf/fapi2/src/plat/plat_error.C

```diff
@@ -47,7 +47,8 @@ void process_HW_callout(FFDC &ffdc, const bool deconfRefTarget)
 	for (auto &hwcallout : ffdc.hwp_errorinfo.hwcallouts)
```

Comment:
Commit message: 

> Need to callout the PLANAR for this hwid.

Only planar?

> "HWP_RC = RC_P10_PCI_REFCLOCK_ERR",

Where it is defined? I could not find it in the https://github.com/open-power/pub-ekb/tree/main-p10 - I am missing something?

> "HWP_CDG_TGT_01_GUARD_REQ = false",
> "HWP_CDG_TGT_01_GUARD_TYPE = GARD_Fatal",

Why the `GUARD_TYPE` is Fatal when the `GUARD_REQ` is false? is it expected?

and Do we need the CDG for the processor? I thought we need to figure out the FRU that contains the "PCI_REF_CLOCK" from the hardware callout `physical:sys-0/node-0/proc-0`, right?

Reply:
It is defined here
https://github.com/open-power/pub-ekb/blob/main-p10/chips/p10/procedures/xml/error_info/p10_sbe_chiplet_init_errors.xml#L43.

No proc callout is required for this.

GUARD_TYPE is set by perl script in ekb, by default.

Reply:
> GUARD_TYPE is set by perl script in ekb, by default.

This is something wrong, I think we need to fix it.